express/scripts/utils.js

```diff
@@ -102,6 +102,17 @@ export async function removeIrrelevantSections(area) {
   });
 }
 
+async function overrideMiloBlocks(area) {
```

Comment:
Can you call this overrideMiloColumn? 
We will only be doing this for the column block.